cmd/bucket-metadata-sys.go

```diff
@@ -397,8 +405,9 @@ func (sys *BucketMetadataSys) Init(ctx context.Context, buckets []BucketInfo, ob
 		return nil
 	}
 
-	// Load PolicySys once during boot.
-	return sys.load(ctx, buckets, objAPI)
+	// Load bucket metadata sys in background
+	go logger.LogIf(ctx, sys.load(ctx, buckets, objAPI))
```

Comment:
I don't see how this is able to be consistent?

For this to work `(*BucketMetadataSys).load(...)` should hold the write lock for sys for the entire operation and concurrentLoad shouldn't grab one AFAICT.

Same for the others.

Reply:
There is no reason for it to be consistent, if the data is missing it loaded from the disk.

The point is we don't need to do this during startup in a blocking fashion and background it.

For any calls which require any configuration at that point in time, it simply needs to lookup the disk one time. That's how the code is written already as a fallback.

For the most part it will be quick, it also doesn't penalize users large sets of buckets.